Reference DYALOG_SHELL_SUBPROCESS in ⎕SHELL docs (#604)
Dyalog/documentation: language-reference-guide/docs/system-functions/shell.md

The default for the input stream is

- `(0 'Null')`: Provide no data on standard input.

!!! aix "Dyalog on AIX"

>     To improve the performance of `⎕SHELL` on AIX, the [`DYALOG_SHELL_SUBPROCESS`](../../unix-installation-and-configuration-guide/docs/configuration-parameters/environment-variables.md) configuration parameter has been added.

Reviewer: ```suggestion
    The performance of `⎕SHELL` on AIX can be improved using the [`DYALOG_SHELL_SUBPROCESS`](../../unix-installation-and-configuration-guide/docs/configuration-parameters/environment-variables) configuration parameter.
```

Reviewer: I've tweaked the replacement text.
